tests/ignite/contrib/metrics/test_gpu_info.py

```diff
@@ -27,10 +27,11 @@ def no_site_packages():
 
 
 @pytest.mark.skipif(python_below_36, reason="No pynvml for python < 3.6")
-def test_no_pynvml_package(no_site_packages):
+def test_no_pynvml_package():
 
-    with pytest.raises(RuntimeError, match="This contrib module requires pynvml to be installed."):
-        GpuInfo()
+    with patch.dict("sys.modules", {"pynvml.smi": None}):
```

Comment:
Do you have an idea why no_site_packages is not working anymore now ?

Reply:
Could it be [we are importing `nvidia_smi` from pynvml](https://github.com/pytorch/ignite/blob/7480fed2d0389acb80b2f922e7242aeb091f8acf/ignite/contrib/metrics/gpu_info.py#L39) instead of `import pynvml` ?